cornac/metrics/ranking.py

```diff
@@ -117,7 +120,7 @@ def compute(self, gt_pos, pd_rank, **kwargs):
 
         """
         dcg = self.dcg_score(gt_pos, pd_rank, self.k)
-        idcg = self.dcg_score(gt_pos, np.argsort(gt_pos)[::-1], self.k)
+        idcg = self.dcg_score(gt_pos, np.concatenate([gt_pos, gt_neg]), self.k)
```

Comment:
Do we need to concatenate gt_pos with gt_neg here? Wouldn't it be enough if we just call  self.dcg_score(gt_pos, gt_pos, self.k)